model/githubapp/github_app_auth.go

```diff
@@ -120,26 +120,26 @@ func (g *GithubAppAuth) CreateGitHubSenderInstallationToken(ctx context.Context,
 // CreateInstallationToken creates an installation token for the given
 // owner/repo. This is never cached, and should only be used in scenarios where
 // the token can be revoked at any time.
-func (g *GithubAppAuth) CreateInstallationToken(ctx context.Context, owner, repo string, opts *github.InstallationTokenOptions) (string, error) {
+func (g *GithubAppAuth) CreateInstallationToken(ctx context.Context, owner, repo string, opts *github.InstallationTokenOptions) (string, *github.InstallationPermissions, error) {
```

Comment:
Overall LGTM -- a nice improvement I think would be to change the Token struct to APIToken, and then add a Token struct to be used by these functions, as opposed to returning multiple things (and could have a tiny  BuildFromService that you could use) but up to you if you want to add that here, non blocking

Reply:
I think I'll leave this as a potential future improvement so that we can merge this faster 🙏🏻